layout.html

```diff
@@ -115,6 +115,11 @@
           <!-- STARTS .content SECTION-->
           <section id="content" class="row">
             {# PAGE CONTENT #}
+	    {% if is_release %}
+	    <div class="wy-alert wy-alert-danger">
+		    This is an old version of the documentation ({{ version }})! <a href="/{{ pagename }}.html">Click here to read the latest version (master)</a>.
```

Comment:
How about:

    You're reading archived documentation for Zephyr version {{ version }}, and not
    <a href="/{{ pagename }}.html">the latest (master) documentation</a>.